observable.go

```diff
@@ -1,14 +1,12 @@
 package rxgo
 
 import (
-	"sync"
-	"time"
-
 	"github.com/reactivex/rxgo/errors"
 	"github.com/reactivex/rxgo/fx"
 	"github.com/reactivex/rxgo/handlers"
 	"github.com/reactivex/rxgo/optional"
 	"github.com/reactivex/rxgo/options"
+	"sync"
```

Comment:
run `gofmt`, internals package on top

Reply:
oops not `gofmt` is on `goimports`[1]

[1]https://godoc.org/golang.org/x/tools/cmd/goimports

Reply:
I've just done it. I'm going to run it also for the other PR. Please wait a few minutes before to review them.
Maybe we could also add this `goimports` step as part of the CONTRIBUTING.md file.

Reply:
> I've just done it. I'm going to run it also for the other PR. Please wait a few minutes before to review them.
> Maybe we could also add this `goimports` step as part of the CONTRIBUTING.md file.

you send PR pls?